blockchain/chain/src/store/chain_store.rs

```diff
@@ -301,6 +301,54 @@ where
         }
     }
 
+    pub async fn validate_tipset_checkpoints(
+        &self,
+        from: Arc<Tipset>,
+        to: ChainEpoch,
```

Comment:
We have to check that the genesis CID actually matches the expected genesis CID so I'm not sure if taking a `to` parameter makes sense. We always have to go to epoch 0.

Reply:
`to` is gone